setup.py

```diff
@@ -38,8 +38,15 @@ def get_version():
     name="fapolicy-analyzer",
     version=get_version(),
     packages=find_namespace_packages(
-        include=["fapolicy_analyzer", "fapolicy_analyzer.*"],
-        exclude=["*.tests"],
+        include=[
+            "fapolicy_analyzer",
+            "fapolicy_analyzer.css",
+            "fapolicy_analyzer.glade",
```

Comment:
I think you need to include "fapolicy_analyzer.locale*" in here.

Reply:
There is some weirdness between how `package_data` is mapped and the package search.  I'll add it for clarity, but it appears to be included from the `package_data` entry alone.

Noting for posterity; I saw that the other resources, which are also sorta "package data", can't be excluded from the  `packages` entry.  Is that perhaps because they are "remapped" by globbing from their actual locations to packages under `fapolicy_analyzer`, while the locale is not (it uses the absolute path rather than a glob)?  Idk.

As long as it works I'm not concerned right now, though someday we should survey other projects to determine if there is some tweaks we could make to these mappings for correctness / clarity.